pkg/cloud/volume_limits.go

```diff
@@ -489,3 +495,58 @@ var nvmeInstanceStoreVolumes = map[string]int{
 	"z1d.metal":       2,
 	"z1d.xlarge":      1,
 }
+
+// / https://aws.amazon.com/ec2/instance-types
+var gpuInstanceGpus = map[string]int{
```

Comment:
In a future PR, it would be nice if we could auto-generate these to prevent operator error (perhaps make a new makefile `pre-release` target that also updates sidecars). 

Here is the magic CLI incantation I came up that worked to auto-gen these locally for me if you want to alley-oop it. But I don't mind making the followup PR during the v1.34 release if you're swamped this sprint. 

If you wrap the actual map with `// CODEGEN_GPU START` and ``CODEGEN_GPU STOP` e.g.

```
var gpuInstanceGpus = map[string]int{
// CODEGEN_GPU START
...
// CODEGEN_GPU STOP
}
```

Then the following incantation when run with gnu-sed will update the `volume_limits.go` file in-place

```
ROOT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../../"
BIN="$ROOT_DIR/bin"
VOLUME_LIMITS_FILE="$ROOT_DIR/pkg/cloud/volume_limits.go"

...

export GPU_LIMITS
GPU_LIMITS=$(get_all_gpus | sort | uniq)
sed -n -e "1,/CODEGEN_GPU START/ p" -e "/CODEGEN_GPU STOP/,$ p" "$VOLUME_LIMITS_FILE" -i
sed '/CODEGEN_GPU\ START/a $GPU_LIMITS' "$VOLUME_LIMITS_FILE" -i
envsubst < "$VOLUME_LIMITS_FILE" | sponge "$VOLUME_LIMITS_FILE"
```

^^ does the following:

1. Store the de-duplicated results of `get_all_gpus` in an exported environment variable
2. Deletes all text between the CODEGEN_GPU START & STOP comments
3. Adds the `$GPU_LIMITS` environment between the START and STOP comments which will be...
4. Substituted with the actual list of GPU Limits and written to `volume_limits.go` 

Note that a few things need to be cleaned up before PR (e.g. making sure the limits get pasted with appropriate tabbing) but it's good enough for me to use next release.  

Hope this was helpful

Reply:
Suggest we hold off on this for now as this table will likely be dropped in the near future

Reply:
I agree with Connor here that we may want to hold off for now.However I  will add a ticket on the backlog to do this though as it will be a good improvement